docs/source/demos/heartbeat_demo.rst

```diff
@@ -55,10 +55,19 @@ The actors in this example can be mapped naturally to :doc:`Characters </api/nuc
   This is modelled by the :class:`~nucypher.characters.lawful.Bob` character.
 
 Install NuCypher
------------------
+----------------
+
+Acquire the ``nucypher`` application code and install the dependencies:
```

Comment:
Should we just link to the installation guide rather than reproducing the instructions here?